l10n/en/firefox/nightly/whatsnew.ftl

```diff
@@ -25,6 +25,7 @@ nightly-whatsnew-if-you-want-to-v3 = If you want to know what’s happening arou
 # Variables:
 #   $attrs (string) - link href and additional attributes
 nightly-whatsnew-want-to-know-which-v2 = Want to know which platform features you could test on { -brand-name-nightly } and can’t see yet on other { -brand-name-firefox } channels? Then have a look at the <a { $attrs }>Nightly Experiments</a> preferences page.
+nightly-whatsnew-want-to-know-which-v3 = Want to know which platform features you could test on { -brand-name-nightly } and can’t see yet on other { -brand-name-firefox } channels? Then have a look at the <a { $attrs }>{ -brand-name-firefox-labs }</a> preferences page.
```

Comment:
This new string needs a comment to explain the variable (you can copy and paste the comment from the v2 string)